package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "waterdataui-assets",
+  "version": "1.0.0",
+  "description": "Compiled static assets for Water Data For The Nation",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "rm -rf dist && mkdir dist",
+    "build": "npm run clean && run-p build:*",
+    "build:css": "node-sass --include-path node_modules/uswds/src/stylesheets assets/styles/main.scss | uglifycss > dist/main.css",
```

Comment:
Does this imply that tools like node-sass have been installed globally? That will not be the case particularly when using Jenkins.

Reply:
It's the locally installed version. I believe the way it works with run scripts is that node adds the bin directory of each package in node_modules/ to the path.

Reply:
In the past that wasn't true but a lot has changed with node since I tried that. As long as it works, I'm happy.